### DISCUSSION TITLE
Nvidia shield record button - Button layout

### DISCUSSION THREAD

**Discussion by Sagi-bb** [PRProd/HA-Firemote#384] [Q&A] [answered]
I would like to change the Button Overrides for the button below looks like a record button .
I cant find in the list a record button.
How can I know which button to change using the Button Overrides codes provided ?

![image](https://github.com/PRProd/HA-Firemote/assets/68748153/a6f1cf19-8e00-422a-b4d9-e58ff0883cad)

**Comment by PRProd** [comment, marked as answer]
Greetings @Sagi-bb!

That button is named ``home-button`` since it's default function is to return the user to the home screen.

**Reply by Sagi-bb** [reply]
ok thank you very much
